packages/components/src/site-thumbnail/use-mshots-img.tsx

```diff
@@ -99,8 +105,8 @@ export const useMshotsImg = (
 		imgProps: {
 			onLoad,
 			onError,
+			srcSet,
 			src: mshotUrl,
-			srcSet: ! isLoading ? srcSet : '',
```

Comment:
@danielbachhuber This fix the issue where the image request happens twice because of how we set the srcset here.